Question1.py

```diff
@@ -0,0 +1,43 @@
+#Ex 8
+
+#Open files to read and write
+
+import re
+vcffile = open("Cflorida.vcf","r")
+outfile = open("CfloridaCounts.txt","w")
+
+#assign regex to variable names
+
+TXnames=r"[Cc][Ff](07)?\.[Aa](2)?"
+FLnames=r"[Cc][Ff]\.[Gg][Aa2](Ii)?"

```

Comment:
r"[Cc][Ff]\.[Gg][Aa2](Ii)?" could just substitute cf.gai.021 to Cf.Gaii.021

So you want to use r"[Cc][Ff]\.(G2|GAI|gai)" to replace the whole Gai part.